src/brainbrowser/surface-viewer/modules/rendering.js

```diff
@@ -623,6 +631,99 @@ BrainBrowser.SurfaceViewer.modules.rendering = function(viewer) {
     return vertex_data;
   };
 
+
+  /**
+  * @doc function
+  * @name viewer.rendering:modelCentric
+  * @param {boolean} if true, recenter all userData shape on origin. Otherwise
+  * return to the original userData.
+  *
+  * @description
+  * Use to recenter data when userData input is shifted in space.
+  *
+  *
+  * ```js
+  * viewer.modelCentric(true);
+  * ```
+  */
+  viewer.modelCentric = function(model_centric) {
+    if (model_centric === undefined) {model_centric = false;}
```

Comment:
Sorry if this is overly pedantic, but I really prefer to keep statements on separate lines:
```
if (model_centric === undefined) {
   model_centric = false;
}
```